### PR TITLE
feat: Create nestjs project structure

### PR DESCRIPTION
**Pull request by thanhtb-2411**
# Summary

# Description

# Checklist

- [ ] 1. Build successfully on local
- [ ] 2. Format all code files included in the PR
- [ ] 3. No more unconfirmed specs in the PR
- [ ] 4. Tested the normal case and error cases following specs
- [ ] 5. Is there a refactor to the old code? If yes, have you tested the old function and noted the impact yet?
- [ ] 6. Loops have a set length and correct termination conditions
- [ ] 7. There aren't any redundant or duplicate codes in PR (clean code)
- [ ] 8. There aren't any hard-coded in the code in PR
- [ ] 9. Optimization logic handling and optimizations SQL (for example, don't get redundant data or add an index to optimize performance, query only necessary fields...)
- [ ] 10. Update issue

# Evidence
(Screenshot or Video)

# Discussion
Note the part that you are not sure about. Comment on your code is OK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# nodejs-base-projects
-NodeJS base projects
+<p align="center">
+  <a href="/">
+    <img src="https://avatars.githubusercontent.com/u/105472712?s=72&v=4" alt="CEV Portal API" width=72 height=72>
```
Comment (khuongln-1346) [inline README.md:3]: @thanhtb-2411 
```suggestion
    <img src="https://avatars.githubusercontent.com/u/105472712?s=72&v=4" alt="NestJS boilerplate" width=72 height=72>
```

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (khuongln-1346) [inline package-lock.json:1]: @thanhtb-2411 Xóa file này đi em nhé

File: README.md (hunk 1 above)
Comment (thanhtb-2411) [reply README.md:3]: Done

**Review by lamngockhuong — APPROVED** [review]
LGTM

**Pull request merged**